### ISSUE TITLE
Vimium can't click X on certain popups.

### ISSUE THREAD

**Issue by zurbrugc** [philc/vimium#3336]
**Chrome version:**
Google Chrome | 74.0.3729.169 (Official Build) (64-bit) (cohort: Stable)
-- | --
Revision | 78e4f8db3ce38f6c26cf56eed7ae9b331fc67ada-refs/branch-heads/3729@{#1013}

**Vimium version:** 1.64.6

Pop-ups on many websites have little x's in the corner to close them. However, Vimium often can't find them and click on them.

Two examples:
![g4gpopup](https://user-images.githubusercontent.com/40577023/58439532-6b43ef80-8089-11e9-90e6-204ebe8660ab.PNG)

![usatodaypopup](https://user-images.githubusercontent.com/40577023/58439536-70a13a00-8089-11e9-8c5c-3262633c89ec.PNG)

To see for yourself, go to any page at the site GeeksForGeeks. Make sure you're not signed into the site, and clear any site cookies. After some browsing, a popup will appear encouraging you to sign in. No amount of "gf" and "gF" will allow me to click on the x when I press "f" or "F".

As a counterpoint, here's an example of where the x is selected:
![githubpopup](https://user-images.githubusercontent.com/40577023/58439699-603d8f00-808a-11e9-9f8e-80aa2d076f44.PNG)

Thanks for looking into this.

**Comment by jk5228** [comment]
Checking the GeekForGeeks site login modal in Chrome inspector, I see that as suspected the close icon is implemented as a span element. I don't know how Vimium generates the list of hints other than what I gathered from briefly skimming the link_hints.coffee file but judging from the complexity and obvious signs that [the hint generation code is pretty heuristic](https://github.com/philc/vimium/blob/881a6fdc3644f55fc02ad56454203f654cc76618/content_scripts/link_hints.coffee#L613-L736) (i.e., not perfect), this issue probably isn't actionable. We could modify the heuristics used to generate the hint list but because they're mere heuristics, increasing the recall rate of the getVisibleClickable function will risk also decreasing its precision, labeling elements that aren't actually clickable.

**Comment by gdh1995** [comment]
Yes, the close button in GeekForGeeks is only `<span class="close">×</span>`, and has no click listeners. So Vimium can not know it's clickable.

It seems that the only way to support such a case is to hint all nodes with a class name matching `/\b[Cc]lose\b/`. I think mistaken hints are somehow better than lacking hints, so my customized version of Vimium (I name it Vimium C) has used this for quite some days.
